.github/workflows/notify_test_workflow.yml

```diff
@@ -119,13 +138,13 @@ jobs:
                 status: status,
                 output: {
                   title: 'Test results',
-                  summary: '[See test results](' + runUrl + ')',
+                  summary: '[See test results](' + check_run_url + ')',
                   text: JSON.stringify({
                     owner: context.payload.pull_request.head.repo.owner.login,
                     repo: context.payload.pull_request.head.repo.name,
-                    run_id: runID
+                    run_id: run_id
                   })
                 },
-                details_url: runUrl,
+                details_url: actions_url,
```

Comment:
Actually I think we can even remove this but let me keep it as is for now (https://github.community/t/create-a-check-run-details-url-is-not-being-set/166002). This is currently just a metadata for now.